src/test/java/test/failedreporter/FailedReporterParametersTest.java

```diff
@@ -77,9 +77,6 @@ public static void runAssertions(File outputDir, String expectedFormat, String[]
       List<String> resultLines = Lists.newArrayList();
       grep(failed, String.format(expectedFormat, expectedKey, expectedKey + "Value"), resultLines);
       int expectedSize = 1;
-      if ("testParam".equals(expectedKey)) {
-        expectedSize = 2;
-      }
```

Comment:
@cbeust : yes this change is part of the pull request.

*Explanation*
Without the fix, `testng-failed.xml` was generated such that the the parameters 
`<parameter name="testParam" value="testParamValue"/>` was being made available
at both the `<suite>` level and `<test>` level.

After I fixed it, it will now be *ONLY* available at the scope as per the original suite file (so in our case only at the `<test>` level).

The reason I deleted these lines from the test was because the tests were doing an incorrect assertion (which I fixed )

*Suite file generated by this unit test*
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite verbose="0" name="Default Suite">
  <parameter name="suiteParam" value="suiteParamValue"/>
  <test name="Command line test 9deb29f4-4649-4fb0-bc67-03fb90c09b5f">
    <parameter name="testParam" value="testParamValue"/>
    <classes>
      <class name="">
        <parameter name="classParam" value="classParamValue"/>
        <methods>
          <include name="f2">
            <parameter name="methodParam" value="methodParamValue"/>
          </include> <!-- f2 -->
        </methods>
      </class> <!--  -->
    </classes>
  </test> <!-- Command line test 9deb29f4-4649-4fb0-bc67-03fb90c09b5f -->
</suite> <!-- Default Suite -->
```

*`testng-failed.xml` generated by the test without the file* (as you can see the `testParam` is available at both the `<suite>` and `<test>` level even though it wasn't the case in the original xml)
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite verbose="0" name="Failed suite [Default Suite]">
  <parameter name="suiteParam" value="suiteParamValue"/>
  <parameter name="testParam" value="testParamValue"/>
  <test name="Command line test 9deb29f4-4649-4fb0-bc67-03fb90c09b5f(failed)">
    <parameter name="testParam" value="testParamValue"/>
    <classes>
      <class name="test.failedreporter.FailedReporterSampleTest">
        <parameter name="classParam" value="classParamValue"/>
        <methods>
          <include name="f2">
            <parameter name="methodParam" value="methodParamValue"/>
          </include> <!-- f2 -->
        </methods>
      </class> <!-- test.failedreporter.FailedReporterSampleTest -->
    </classes>
  </test> <!-- Command line test 9deb29f4-4649-4fb0-bc67-03fb90c09b5f(failed) -->
</suite> <!-- Failed suite [Default Suite] -->
```